spot-client/src/common/remote-control/xmpp-connection.js

```diff
@@ -434,6 +434,6 @@ export default class XmppConnection {
      * @returns {void}
      */
     _sendPresence() {
-        this.room.sendPresence();
+        this.room && this.room.sendPresence();
```

Comment:
Were you hitting errors that needed these truthy checks? I can see there being problems in general but am curious what you were seeing.

Reply:
Yes, I need to double check when exactly this was happening. I think it's when the joinCode is set and the room is not ready yet.